content/blog/2023/2023-08-24-fairness-reweighing-dataset.md

```diff
@@ -0,0 +1,41 @@
+title = "Orange Fairness - Reweighing a Dataset"
+type = "blog"
+thumbImage = "/blog_img/2023/2023-08-24-fairness-reweighing-dataset-thumb.png"
+frontPageImage = "/blog_img/2023/2023-08-24-fairness-reweighing-dataset-thumb.png"
+blog = ["fairness", "reweighing"]
+shortExcerpt = "Introducing the Reweighing widget in Orange, a solution for dataset bias mitigation."
+longExcerpt = "Building on our exploration of the Orange fairness addon, this blog delves into the Reweighing widget. By adjusting weights for dataset instances, the widget addresses bias, focusing on underrepresented groups. Using the Compas dataset as an example, we demonstrate how bias decreases post-reweighting, presenting visual insights into the distribution of adjusted weights and their impact on dataset fairness."
+
++++
+
+In the [previous blog post]({{< ref "/blog/2023/2023-08-23-fairness-dataset-bias.md" >}}), we introduced the Orange fairness addon along with the Dataset Bias and As Fairness widgets. We also demonstrated how to use them to detect bias in a dataset and visualized the results for a better understanding. In this blog, we will introduce the Reweighing widget, which we can use to mitigate bias in a dataset, resulting in fairer machine learning models learning from it.
+
+### Reweighing:
+
+The Reweighing widget offers a solution to mitigate bias in datasets by assigning weights to individual instances. These weights are determined using the [Reweighing algorithm](https://link.springer.com/article/10.1007/s10115-011-0463-8). Essentially, the algorithm gives higher weights to underrepresented pairs of protected attributes and classes and reduces weights for overrepresented ones. This strategy encourages the model to prioritize learning from underrepresented groups while de-emphasizing overrepresented groups. In Orange, we can use the Reweighing widget in two ways:
+
+- **Reweighing a dataset:** The widget can be used to reweigh a dataset, which we can then use to train a model. Simply connect your dataset to the widget, and the output will be a dataset with appropriately weighted instances.
+
+- **Reweighing as a preprocessor:** The widget can function as a preprocessor for a specific model. By connecting the Reweighing widget directly to your chosen model, the dataset is automatically reweighed prior to the training phase.
+
+## Orange use case
+
+Now that we have grasped how the Reweighing widget functions and its applications, let us explore a real-world example of using it to assign weights to data. For this illustration, we will use the [Compas dataset](https://github.com/propublica/compas-analysis). Contrary to the example in the previous blog, we will not use the As Fairness widget to select fairness attributes. This is because datasets with a fairness tag come with default fairness attributes. Specifically, for the Compas dataset, "race" is identified as the protected attribute, with "Caucasian" set as the privileged, protected attribute value.
```

Comment:
You should shortly describe what the data set is about and what are we predicting. Otherwise users can not understand the interpretation.

I know you have a link, but one individual blog should be most self-contained.